cmd/operator/main.go

```diff
@@ -251,25 +251,31 @@ func printHealthCriteria(logger *logrus.Logger, healthCriteria []config.HealthCr
 	}
 }
 
-func determineDefaultProject() (string, error) {
+func determineProjectID() (string, error) {
 	if metadata.OnGCE() {
 		v, err := metadata.ProjectID()
 		if err != nil {
-			return "", errors.Wrapf(err, "error when getting project ID from compute metadata: %v", err)
+			return "", errors.Wrapf(err, "error when getting project ID from compute metadata")
 		}
 		return v, nil
 	}
 
 	// Try to get project ID by retrieving default value in gcloud.
-	cmd := exec.Command("gcloud", "config", "get-value", "core/project")
-	var out bytes.Buffer
-	cmd.Stdout = &out
+	cmd := exec.Command("gcloud", "config", "get-value", "core/project", "-q")
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
```

Comment:
not necessary but you can do this in the future:

var stdout, stderr bytes.Buffer
cmd.Stdout, cmd.Stderr = &stdout, &stderr

Reply:
I made this change too